### PR TITLE
Jinja Cheat Sheet

### PR DESCRIPTION
**Pull request by ayush1095**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.
- [x] New Instant Answer
  - [x] Cheat Sheets: **`New {Jinja} Cheat Sheet`**
  - [ ] Other: **`New {IA Name} Instant Answer`**
- [ ] Improvement
  - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
  - [ ] Enhancement: **`{IA Name}: {Description of Improvements}`**
- [ ] Non–Instant Answer
  - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**
###### Description of changes

This PR adds Jinja Cheat Sheet.
###### Which issues (if any) does this fix?

None
###### People to notify (@mention interested parties)

@zekiel @moollaza 

---

Instant Answer Page: https://duck.co/ia/view/jinja_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@ayush1095 Thanks for the pull request! Unfortunately, we're only accepting contributions and improvements to a limited portion of the search space.
            
The staff and community are currently focused on making DuckDuckGo the [best search engine for programmers](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53). To ensure we're making as much progress as possible, **we are only accepting Instant Answers and updates related to the programming mission**.
    
If this is not related to the Programming Mission, it will likely be closed but you can:
    
1. Help us prevent further confusion by telling us where you found out about DuckDuckHack or where we could've alerted you earlier.
2. Keep this code handy! We'll be expanding into other search topics in the future!
3. Join us at our new home: https://forum.duckduckhack.com/

If this is related to the Programming Mission, it will be reviewed by staff and the community as soon as possible!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/jinja.json [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+                "key": "first(seq)",
+                "val": "Return the first item of a sequence"
+            },
+            {
+                "key": "groupby(value, attribute)",
+                "val": "Group a sequence of objects by a common attribute"
+            },
+            {
+                "key": "sort(value, reverse=False, case_sensitive=False, attribute=None)",
+                "val": "Sort an iterable.Default is ascending"
```
Comment (adityasiwan) [inline share/goodie/cheat_sheets/json/jinja.json]: Lets add a space after the period

File: share/goodie/cheat_sheets/json/jinja.json [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                "key": "{% ... %}",
+                "val": "For control statements"
+            },
+            {
+                "key": "{{ ... }}",
+                "val": "For expressions to print to the template output"
+            },
+            {
+                "key": "#  ... ##",
+                "val": "For line Statements"
```
Comment (adityasiwan) [inline share/goodie/cheat_sheets/json/jinja.json]: Lets change the `Statements` to lowercase 😄

File: share/goodie/cheat_sheets/json/jinja.json [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+                "key": "+",
+                "val": "Adds two objects together"
+            },
+            {
+                "key": "-",
+                "val": "Subtract the second number from the first one"
+            },
+            {
+                "key": "/",
+                "val": "Divide two nubers and returns a floating point number"
```
Comment (adityasiwan) [inline share/goodie/cheat_sheets/json/jinja.json]: Fix the typo here for `nubers`

**Comment by ayush1095** [comment]
@gautamkrishnar I have implemented the recommendations as mentioned by @adityasiwan . Please review the changes. TIA 😄

**Comment by daxtheduck** [comment]
@ayush1095 this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by gautamkrishnar** [comment]
@ayush1095 thanks for contributing...

**Comment by ayush1095** [comment]
@gautamkrishnar The  [beta version](https://beta.duckduckgo.com/?q=jinja+cheat+sheet&t=h_&ia=answer&iax=1)  is not esacaping the `{ }` curly braces correctly. Please recommend how this can be done. 😄

**Comment by gautamkrishnar** [comment]
As per as the page: https://docs.duckduckhack.com/frontend-reference/cheat-sheet-reference.html
You need to escape it as `\\{` and `\\}`. Please open a PR ASAP to fix it, so that i can merge it.

**Comment by ayush1095** [comment]
thanks for the help 😄 I have opened a PR. Please review the changes.
@gautamkrishnar

**Pull request merged**